arch/arm/configs/miyoo_defconfig

```diff
@@ -80,11 +83,6 @@ CONFIG_SND_SOC=y
 CONFIG_SND_SUNIV_CODEC=y
 CONFIG_USB_DYNAMIC_MINORS=y
 CONFIG_USB_OTG=y
-CONFIG_USB_EHCI_HCD=y
-CONFIG_USB_EHCI_HCD_PLATFORM=y
-CONFIG_USB_OHCI_HCD=y
-CONFIG_USB_OHCI_HCD_PLATFORM=y
-CONFIG_USB_STORAGE=y
```

Comment:
we could add support for reading /media at some point?

Reply:
This driver is to connect usb drive to handheld, not to share

Reply:
yes that I understand, but do you imply we can't read external storage from handheld? I see connected USB drive in `lsdisk` but couldn't mount without arg error.

Reply:
Then why rm this config? I don't follow.

Reply:
You said that is not working. Also to free memory, umtprd sometimes crash with oom on old kernel

Reply:
Ok umtprd is mega convenient, thanks for explanation. Lets keep usb ex. drives in a backlog for e.g. swaping files (.e.g. savestates) between different SD cards.